README.md

```diff
@@ -1 +1,283 @@
-# backend
+
+# hot-coffee-sql
+
+## Learning Objectives
+
+- SQL
+- PostgreSQL
```

Comment:
I think this project also needs to show the difference between SQLite and PostgeSQL as databases and to do so, it should require such things, that is featured only in PostgreSQL, like it's scalability, concurrent queries, unique data types, range of tools and etc.